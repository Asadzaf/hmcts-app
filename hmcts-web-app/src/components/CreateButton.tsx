const CreateButton = () => {
    return (
        <button type="submit" className="govuk-button" data-module="govuk-button">
            Save and continue
        </button>
    )
}

export default CreateButton