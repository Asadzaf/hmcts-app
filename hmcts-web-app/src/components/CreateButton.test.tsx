import '@testing-library/jest-dom';

import { render } from '@testing-library/react';
import CreateButton from './CreateButton';

test('renders Header correctly', () => {
  const { asFragment } = render(<CreateButton/>);
  expect(asFragment()).toMatchSnapshot();
});
