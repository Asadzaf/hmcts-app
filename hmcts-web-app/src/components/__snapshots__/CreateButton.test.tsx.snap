// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly 1`] = `
<DocumentFragment>
  <button
    class="govuk-button"
    data-module="govuk-button"
    type="submit"
  >
    Save and continue
  </button>
</DocumentFragment>
`;
